Fix the description of chart.commonSeriesSettings.point.visible (T970959) (#1860)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/visible.md

dxChartSeriesTypes.LineSeries,dxChartSeriesTypes.StackedLineSeries,dxChartSeriesTypes.FullStackedLineSeries,dxChartSeriesTypes.StackedSplineSeries,dxChartSeriesTypes.FullStackedSplineSeries,dxChartSeriesTypes.SplineSeries,dxChartSeriesTypes.StepLineSeries,dxChartSeriesTypes.AreaSeries,dxChartSeriesTypes.StackedAreaSeries,dxChartSeriesTypes.FullStackedAreaSeries,dxChartSeriesTypes.SplineAreaSeries,dxChartSeriesTypes.StepAreaSeries,dxChartSeriesTypes.RangeAreaSeries,dxChartSeriesTypes.ScatterSeries

---

> This property's default value is **false** for series of *area types. This requires that you set this option to **true** manually to display series points.

Reviewer: ```suggestion
This property's default value is **false** for series of *area* types. Set this option to **true** to display points in these series types.
```
